src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml

```diff
@@ -205,7 +204,7 @@
                 DockPanel.Dock="Top"
                 Height="55"
                 Margin="1"
-                Visibility="{Binding Path=IsPendingRestart, Mode=OneWay, Converter={StaticResource BoolToVis} }">
```

Comment:
have you used the wpf live tree to verify that this dockPanel actually has a dataContext that is not null - you may need to set it explicitly, not on the parent, but on this sub tree.

Reply:
Yes, I did that. The data context is inherited from parent

Reply:
@QilongTang you are missing `RaisePropertyChanged("IsPendingRestart"); or `nameOf(IsPendingRestart)` inside your setter for the public property.